main.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Please avoid disabling ESlint to get around linter warnings problems. These ESlint warnings are there to ensure you adhere to correct coding standards and when you have warnings they usually signal an underlying issue with how you've written your code.

Reply:
OK, thanks I'm just wondering about calling back a function before it is created so if I do that eslint make an error and that that's the function that you said to change ...

Reply:
make sure to declare the function before you use it. that's what the ESlint error is referring to.